fvtr/python/python.exp

```diff
@@ -105,6 +105,10 @@ set env(TMPDIR) "$FULLPATH/tmp[pid]"
 exec mkdir $env(TMPDIR)
 printit "TMPDIR=\"$env(TMPDIR)\""
 
+# Test for the issue #45
+printit "Executing sqlite3 importing test."
+exec $env(AT_DEST)/bin/python3 -c "import sqlite3"
+
```

Comment:
According Python comments on sqlite test case (/Lib/test/test_sqlite.py):

import test.support

**# Skip test if _sqlite3 module not installed
test.support.import_module('_sqlite3')**

import unittest
import sqlite3
from sqlite3.test import (dbapi, types, userfunctions,
                                factory, transactions, hooks, regression,
                                dump)

def load_tests(*args):
    if test.support.verbose:
        print("test_sqlite: testing with version",
              "{!r}, sqlite_version {!r}".format(sqlite3.version,
                                                 sqlite3.sqlite_version))
    return unittest.TestSuite([dbapi.suite(), types.suite(),
                               userfunctions.suite(),
                               factory.suite(), transactions.suite(),
                               hooks.suite(), regression.suite(),
                               dump.suite()])

if __name__ == "__main__":
    unittest.main()


As we are now explicitly adding it, this test may run. So we don't need to worry about the test execution. Only move the test inside the python3 IF.